lib/DDG/Spice/Gulp.pm

```diff
@@ -0,0 +1,16 @@
+package DDG::Spice::Gulp;
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice wrap_jsonp_callback => 1;
+spice to => 'http://npmsearch.com/query?fields=name,keywords,rating,description,author,modified,homepage,version,license&q=$1%20AND%20keywords:gulpplugin&sort=rating:desc';
+
+triggers any => 'gulp plugin', 'gulp plugins', 'gulpjs', 'gulpjs plugin', 'gulpjs plugins', 'gulp.js', 'gulp.js plugin', 'gulp.js plugins';
```

Comment:
@moollaza can't seem to get it work. I could do something like `map { ("$_ plugin", "$_ plugins") } qw (gulp gulpjs gulp.js);`, but I think you'd have to flatten the array after so might not be worth it.